mmcv/ops/csrc/pytorch/cpu/sigmoid_focal_loss.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright(c) OpenMMLab.All rights reserved.
+#include <cfloat>
+#include <cmath>
+#include "pytorch_cpp_helper.hpp"
+#include "pytorch_device_registry.hpp"
+
+template <typename T>
+void sigmoid_focal_loss_forward_cpu_kernel(const int N, const T* input, const int64_t* target,
+                               const T* weight, T* output,
+                               const float gamma,
+                               const float alpha,
+                               const int num_classes) {
+    for (int i = 0; i < N; i++) {
+        T p = (T) 1. / ((T)1. + expf(-input[i]));
```

Comment:
expf can be replaced with exp since c++ has overloaded it.

Reply:
OK. I would fix this in next commit.